src/components/ReportingConfig/_ReportingConfig.scss

```diff
@@ -0,0 +1,3 @@
+.justify-left {
+  margin-left: -40px;
+}
```

Comment:
nit: bit wary of a seemingly-generic class name that is specific to this use case, especially when it's not namespaced. For example, I believe if some other page happen to start using `.justify-left` because they think its a Bootstrap class name, these styles might start applying to them.

I would recommend namespacing `.justify-left` under a class name specific to this use case, e.g. `.reporting-config .justify-left` or similar.